sway/tree/container.c

```diff
@@ -1049,8 +1049,7 @@ static void container_fullscreen_workspace(struct sway_container *con) {
 		return;
 	}
 	bool enable = true;
-	set_fullscreen_iterator(con, &enable);
-	container_for_each_child(con, set_fullscreen_iterator, &enable);
+	set_fullscreen(con, &enable);
```

Comment:
We no longer need a data pointer for the iterator, so I think just `set_fullscreen(con, true)` is cleaner.